### ISSUE TITLE
Filter by class time

### ISSUE THREAD

**Issue by epicfaace** [theopolisme/fast-courses#2]
That would be useful to see which classes are in the morning, etc. If you have a timeslot that's free and want to find a class that's in that time, this feature would also help.

**Comment by theopolisme** [comment]
Agreed, this would be great! We'd just need to find/build a time-selector widget. Any recommendations?

**Comment by epicfaace** [comment]
I don't know if we'd necessarily need a time-selector widget -- just get the list of distinct times for all the classes, and display them as checkboxes as in all the other filters.

**Comment by theopolisme** [comment]
Hmm, I feel like that might get kinda unwieldly. Seems a better UX might be "I'm looking for classes falling between x and y, show me everything"?
